doc/changelog/10-standard-library/12121-master+fix11903-warn-non-truly-fixpoint.rst

```diff
@@ -0,0 +1,11 @@
+- **Fixed:**
+  :cmd:`Fixpoint`s of the standard library without a recursive call turned
+  into ordinary :cmd:`Definition`s
```

Comment:
I believe that the compilation error in the refman is due to this:
```suggestion
  :cmd:`Fixpoint`\s of the standard library without a recursive call turned
  into ordinary :cmd:`Definition`\s
```

Reply:
OK, I did two entries.